Source/CesiumRuntime/Public/CesiumGeoJsonDocument.h

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020-2025 CesiumGS, Inc. and Contributors
+
+#pragma once
+
+#include "CesiumGeoJsonObject.h"
+#include "CesiumUtility/IntrusivePointer.h"
+#include "CesiumVectorData/GeoJsonDocument.h"
+#include "Kismet/BlueprintAsyncActionBase.h"
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "Templates/SharedPointer.h"
+#include "UObject/ObjectMacros.h"
+
+#include <memory>
+#include <optional>
+
+#include "CesiumGeoJsonDocument.generated.h"
+
+/**
+ * @brief A vector document containing a tree of `FCesiumGeoJsonObject` values.
+ */
+USTRUCT(BlueprintType)
+struct FCesiumGeoJsonDocument {
+  GENERATED_BODY()
+
+  /**
+   * @brief Creates an empty `FCesiumGeoJsonDocument`.
+   */
+  FCesiumGeoJsonDocument() : _document(nullptr) {}
+
+  /**
+   * @brief Creates a `FCesiumGeoJsonDocument` wrapping the provided
+   * `CesiumVectorData::GeoJsonDocument`.
+   */
+  FCesiumGeoJsonDocument(
+      std::shared_ptr<CesiumVectorData::GeoJsonDocument>&& document)
+      : _document(std::move(document)) {}
+
+private:
+  std::shared_ptr<CesiumVectorData::GeoJsonDocument> _document;
+
+  friend class UCesiumGeoJsonDocumentBlueprintLibrary;
+};
+
+/**
+ * @brief A Blueprint Function Library providing functions for interacting with
+ * a `FCesiumVectorDocument`.
+ */
+UCLASS()
+class UCesiumGeoJsonDocumentBlueprintLibrary
+    : public UBlueprintFunctionLibrary {
+  GENERATED_BODY()
+
+public:
+  /**
+   * @brief Attempts to load a `FCesiumVectorDocument` from a string containing
```

Comment:
There's more instances of "Vector" instead of GeoJSON; I'd do a find & replace throughout the docs